Nuro/Features/Child/Routine/Controllers/ChildRoutineViewController+DataSource.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  ChildRoutineViewController+DataSource.swift
+//  Nuro
+//
+//  Created by Karen Natalia on 05/10/22.
+//
+
+import UIKit
+
+extension ChildRoutineViewController: UICollectionViewDataSource {
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        viewModel.activityName.count
+    }
+    
+    func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        let cell = collectionView.dequeueReusableCell(withReuseIdentifier: ChildActivityCollectionViewCell.identifier, for: indexPath) as! ChildActivityCollectionViewCell
+        
+        if indexPath.row == 0 && hideFirstActivityCard == true {
```

Comment:
try giving a boolean name a more actionable name
```suggestion
        if indexPath.row == 0 && isFirstActivityCardHidden {
```